internal/router/interchain.go

```diff
@@ -165,13 +155,11 @@ func (router *InterchainRouter) classify(block *pb.Block) map[string]*pb.MerkleW
 		m[k] = txs
 	}
 
-	target := make(map[string]*pb.MerkleWrapper)
+	target := make(map[string]*pb.InterchainTxWrapper)
 	for dest, txs := range m {
-		wrapper := &pb.MerkleWrapper{
-			BlockHeader:       block.BlockHeader,
+		wrapper := &pb.InterchainTxWrapper{
```

Comment:
Should `InterchainTxWrapper ` add block height?

Reply:
Yes, I am working on that.